charts/furiosa-metrics-exporter/values.yaml

```diff
@@ -22,7 +22,7 @@ daemonSet:
                 values:
                   - "true"
   image:
-    repository: docker.io/furiosaai/furiosa-metrics-exporter
+    repository: registry.corp.furiosa.ai/furiosa/furiosa-metrics-exporter
```

Comment:
image should not be changed. public helm chart should reference public docker image.

Reply:
Reverted at da5f0a41